homeassistant/components/victron_ble/__init__.py

```diff
@@ -0,0 +1,66 @@
+"""The Victron Bluetooth Low Energy integration."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+import logging
+
+from victron_ble_ha_parser import VictronBluetoothDeviceData
+
+from homeassistant.components.bluetooth import (
+    BluetoothScanningMode,
+    async_rediscover_address,
+)
+from homeassistant.components.bluetooth.passive_update_processor import (
+    PassiveBluetoothProcessorCoordinator,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ACCESS_TOKEN, Platform
+from homeassistant.core import HomeAssistant
+
+_LOGGER = logging.getLogger(__name__)
+
+
+type VictronBLEConfigEntry = ConfigEntry[VictronBLEData]
+
+
+@dataclass
+class VictronBLEData:
+    """Class to hold data for the Victron BLE device."""
+
+    coordinator: PassiveBluetoothProcessorCoordinator | None = None
```

Comment:
No need to wrap a single object in a dataclass

Reply:
Ok, so the runtime data will simply be the coordinator then, correct?

Reply:
yes